packages/atomic/src/components/search/result-template-components/result-template-decorators.tsx

```diff
@@ -20,8 +20,7 @@ export class MissingResultParentError extends Error {
  * A [StencilJS property decorator](https://stenciljs.com/) to be used for result template components.
  * This allows the Stencil component to fetch the current result from its rendered parent, the `atomic-result` component.
  *
- *
- * @example
+ * Example:
```

Comment:
We don't generate docs for this anyway, but I thought I'd take care of it in this PR just for uniformity 🙂